.github/workflows/ci.yaml

```diff
@@ -9,6 +9,8 @@ on:
     paths-ignore:
       - "docs/*"
       - ".readthedocs.yaml"
+      - ".github/workflows/cpu-long-tests.yaml"
+      - ".github/workflows/gpu-integration-tests.yaml"
```

Comment:
Adding skips here so that when we make some small change to our action that runs on AWS we don't fire off the whole testing matrix.